test/integration/etcd_storage_path_test.go

```diff
@@ -1091,6 +1116,9 @@ func TestEtcd3StoragePath(t *testing.T) {
 			t.Errorf("empty test data for %s from %s", kind, pkgPath)
 			continue
 		}
+		if testData.namespaceOverride {
```

Comment:
move this into the OriginLegacyKind-handling block above? rest mappings should be correct for all discoverable resources

Reply:
> move this into the OriginLegacyKind-handling block above? rest mappings should be correct for all discoverable resources

ok